src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+@import "../utils/_variables";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 950px;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $white-color;
+  border: 1px solid $border-color;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

Reply:
sounds good, ill do my best to use it in the future. thx a lot